rules/apps.libsonnet

```
@@ -59,21 +59,21 @@
             ||| % $._config,
           },
           {
-            record: 'cluster:namespace:pod_memory:active:kube_pod_container_resource_requests',
+            record: 'cluster:namespace:pod_memory:active:kube_pod_resource_requests',
             expr: |||
-              kube_pod_container_resource_requests{resource="memory",%(kubeStateMetricsSelector)s}  * on (namespace, pod, %(clusterLabel)s)
+              kube_pod_resource_requests{resource="memory",%(kubeStateMetricsSelector)s}  * on (namespace, pod, %(clusterLabel)s)
```

I would expect `kubeSchedulerSelector` instead (or maybe we need to add a new config parameter since the resource metrics are exposed by the /metrics/resources endpoint and not /metrics?).
```suggestion
              kube_pod_resource_requests{resource="memory",%(kubeSchedulerSelector)s}  * on (namespace, pod, %(clusterLabel)s)
```

---

It seems to be `kube_pod_resource_requests` [here](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1748-pod-resource-metrics#expose-new-metrics).

---

the k/k code says different :) 
https://github.com/kubernetes/kubernetes/blob/45df8f0bb3c0165c5a3e716e803402fa0bd7087d/pkg/scheduler/metrics/resources/resources.go#L54-L71